drop-in/src/main/java/com/adyen/checkout/dropin/ui/stored/PreselectedStoredPaymentViewModel.kt

```diff
@@ -66,21 +69,40 @@ class PreselectedStoredPaymentViewModel(
             "payButtonClicked - componentState.isReady: ${componentState?.isReady} - " +
                 "fragmentState: $fragmentState"
         )
+        val componentError = lastComponentError
```

Comment:
Why do we want to store the last error? In case the dialog is being shown when the Fragment gets destroyed?

Reply:
No, the error might happen before the user taps Pay, so we need to store it in order to notify about it if Pay is clicked.

Reply:
Then maybe add a comment or rename the variable to something like `errorBeforePay` to make the use case more clear.

Reply:
Added a comment, let me know if it's still unclear.